packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistFolderNavItem.module.css

```diff
@@ -9,8 +9,12 @@
   margin-right: 8px;
 }
 
-.iconFolderUpdated {
-  color: var(--secondary);
+.iconFolder path {
+  fill: var(--harmony-n-950);
+}
+
+.iconFolderUpdated path {
+  fill: var(--harmony-secondary);
```

Comment:
Interesting. Can we use accent color for icon to cover this?

Reply:
yeah lemme switch it out!